tests/pipert/core/test_shared_memory.py

```diff
@@ -0,0 +1,43 @@
+import pipert.core.multiprocessing_shared_memory as sm
+
+
+class DummySharedMemoryGenerator(sm.MpSharedMemoryGenerator):
+    def __init__(self):
+        super().__init__("dummy_component", max_count=5)
+
+
+def test_cleanup():
+    generator = DummySharedMemoryGenerator()
+    generator.get_next_shared_memory()
+    generator.get_next_shared_memory()
+    generator.cleanup()
+    assert generator.shared_memories == {}
+
+
+def test_get_next_shared_memory():
```

Comment:
this test is not enough.
This function does two things:
1. next name
2. name to unlink

add test for unlink

Reply:
I believe the test: `test_max_count` does that